etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
+    value: "/clusters/gluster/$cluster.id/volumes/$volume.id
```

Comment:
Here gluster doesnt maintain any UUID for cluster but does for volumes and bricks etc. Is it like responsibility of the bridge code to decide how to populate the UUID for an object?